build-tools/packages/build-tools/src/common/utils.ts

```diff
@@ -11,13 +11,18 @@ import * as path from "path";
 import * as util from "util";
 import { pathToFileURL } from "node:url";
 
+/**
+ *	An array of commands that are known to have subcommands and should be parsed as such
+ */
+const multiCommandExecutables = ["flub", "biome"];
+
 export function getExecutableFromCommand(command: string) {
 	let toReturn: string;
 	const commands = command.split(" ");
-	if (commands[0] === "flub") {
+	if (multiCommandExecutables.includes(commands[0])) {
 		// Find the first flag argument, and filter them out. Assumes flags come at the end of the command, and that all
-		// subsequent arguments are flags.
-		const flagsStartIndex = commands.findIndex((c) => c.startsWith("-"));
+		// subsequent arguments are flags. Also treat "." as a flag.
+		const flagsStartIndex = commands.findIndex((c) => c.startsWith("-") || c === ".");
```

Comment:
Seems like we could update this comment to find first option or non-command. Seems like starting with `-` or `.` would qualify. I could see a `./...` argument also being distinguished as a path option and not a command.
Feels better than saying treat "." as a flag as I didn't know if we were finding flags and what the implications would be. After more reading I understand that the real point is to identify the commands, which is command line less the options.